### ISSUE TITLE
track downloads from individual shared links

### ISSUE THREAD

**Issue by jospoortvliet** [nextcloud/files_downloadactivity#25]
We have this nice feature to have a bunch of separate links per user (by entering their mail address) but we don't track who of those users downloaded the files. Would be great if we did!

**Comment by builder-main** [comment]
I totally agree with that. Very useful in case you share a document and want to see who did read it or not.

**Comment by erlendfalch** [comment]
Just want to chip in why this is relevant.

We had a use case where I need (as a startup) share a Virtual Data Room to multiple users/companies. As a part of this, it's crucial to know who (preferably based on label of link) has view and downloaded how files how many times. 

Would have bought a coffee or a beer for you for this function! :)
